### ISSUE TITLE
Add README.txt

### ISSUE THREAD

**Issue by williamchong007** [likecoin/likecoin-wordpress#9]

**Comment by peter279k** [comment]
According to this repository, the `README.md` has been already existed.
Is the `README.md` not good enough?

**Comment by williamchong007** [comment]
Yes I kind of forgot to update this issue when I rushed a simple README.md in here
But it seems kind of too simple. Maybe we should explain what is the features of this plugin, and other useful information in README, to allow simple understanding of what this repo does for visitors
